server/src/module/template/template.module.ts

```diff
@@ -6,7 +7,10 @@ import { TemplateService } from './template.service';
     exports: [TemplateService],
 })
 export class TemplateModule implements OnApplicationBootstrap {
-    constructor(private readonly templateService: TemplateService) {}
+    constructor(
+        private readonly templateService: TemplateService,
+        private readonly orm: MikroORM
```

Comment:
Is the `orm` parameter needed? It's not used inside this module's class.

Reply:
removed with a6260042c02785befaaef06ad6af1fb886b57190